site/src/Librecores/ProjectRepoBundle/DataFixtures/ORM/LoadDevTestData.php

```diff
@@ -96,9 +96,9 @@ public function load(ObjectManager $manager)
         $projectOptimsoc->setSourceRepo($sourcerepoOptimsoc);
         $manager->persist($projectOptimsoc);
 
-        //classificatioin hierarchy
-        foreach ($this->classifiers as $arr) {
-            $classifier =  $this->createClassifier($arr[0], $arr[1]);
+        // classification hierarchy
+        foreach (self::classifiers as $category) {
```

Comment:
Why `self::` ?

Reply:
For accessing the constant of that fixture class named as classifiers.

Reply:
:+1: